docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
+```
+
+```json
+404 Not Found
+
+{"error":"compendia ids not found"}
+```
+
+### Request
+
+`POST /api/v1/substitution`
+
+input of request-body for substitution <br/>
+( where `checked` will be a list of files choosen for substitution )
+```json
+{
+  "user": "user_id",
+  "ercs": [
+    {
+      "id": "base_erc_id",
+      "checked": [],
+      "user": "creator_id"
+    },
+    {
+      "id": "base_erc_id",
+      "checked": [],
+      "user": "creator_id"
+    }
+  ]
+}
+```
+
+### Response
+
+```json
+200 OK
+
+{
+  "user": "user_orcid_id",
+  "ercs": [
+    {
+      "id": "base_erc_id",
+      "checked": [],
+      "user": "creator_id"
+    },
+    {
+      "id": "base_erc_id",
+      "checked": [],
+      "user": "creator_id"
+    }
+  ]
+}
+```
+
```

Comment:
Add here an explanation how you can subsequently use the ID to retrieve the new ERC via the endpoint `/api/v1/compendium` and how that response looks like (don't forget the `substitution: true`).